CONTRIBUTING.md

```diff
@@ -171,3 +169,18 @@ A PR can only be merged into master by a maintainer if:
 
 Any maintainer is allowed to merge a PR if all of these conditions are
 met.
+
+* `bundle exec rake release`
```

Comment:
I just detected this unintended addition, we can force push this commit and use it to see if codeclimate retriggers.